includes/admin/class-wcmp-export.php

```diff
@@ -335,7 +335,7 @@ public function add_shipments(array $order_ids, bool $process)
             $order        = WCX::get_order($order_id);
             $consignment  = (new WCMP_Export_Consignments($order))->getConsignment();
             $extraOptions = WCX_Order::get_meta($order, WCMYPA_Admin::META_SHIPMENT_OPTIONS_EXTRA);
-            $colloAmount  = $extraOptions["collo_amount"] ?? 1;
+            $colloAmount  = (int) $extraOptions["collo_amount"] ?? 1;
```

Comment:
Je moet hier "clarifying parentheses" toepassen.

```php
(int) $extraOptions["collo_amount"] ?? 1;
```
kan betekenen:
```php
((int) $extraOptions["collo_amount"]) ?? 1;
```
Wat, als dat `null` is, `0` wordt. `0 ?? 1 = 0`, want 0 is niet "niks".  

Als je het zo schrijft is er geen verwarring mogelijk en werkt het gegarandeerd zoals je hoopt:
```php
(int) ($extraOptions["collo_amount"]) ?? 1);
```